### ISSUE TITLE
Loss Backpropogation 

### ISSUE THREAD

**Issue by vatsalsaglani** [orobix/Prototypical-Networks-for-Few-shot-Learning-PyTorch#20]
During the learning stage, the loss isn't backpropagated to the model and I am obtaining the same accuracy and loss even after training for a huge number of epochs.

**Comment by dnlcrl** [comment]
Hi @vatsalsaglani, it looks like you aren't accumulating the gradients for the loss, unfortunately we cannot reproduce this behavior, I would suggest to check if the model's parameters tensors do require gradients or to make sure that backward() and optim.setp() are effectively called between each iteration.
